packages/fake-driver/lib/fake-app.js

```diff
@@ -1,13 +1,11 @@
-import B from 'bluebird';
-import fs from 'fs';
+import {readFile} from 'fs/promises';
```

Comment:
readFile with promises could be imported from fs submodule of appium-support

Reply:
I'm curious: why promisify the `fs` api with bluebird?  performance reasons?

Reply:
Before promise versions of the node apis were shipped with Node, we put promisified versions in appium-support since we used them everywhere. I'm not sure why we would keep using them in appium-support though now, unless there are functional differences (@mykola-mokhnach ?)

Reply:
(I've pushed this change)

Reply:
> Before promise versions of the node apis were shipped with Node, we put promisified versions in appium-support since we used them everywhere. I'm not sure why we would keep using them in appium-support though now, unless there are functional differences (@mykola-mokhnach ?)

Exactly. As for appium-support we could now replace bluebird usage with async file operations natively provided by node. I don't see any point in changing method signatures because it would require too many changes in depending modules